ci-operator/config/openshift/release/openshift-release-master__cnv-nightly-4.18.yaml

```
@@ -52,7 +52,8 @@ tests:
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
     env:
-      COMPUTE_NODE_TYPE: c8g.metal-24xl
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: c6g.metal-12xl
```

```suggestion
      COMPUTE_NODE_TYPE: c6g.metal
```
no `c6g.metal-12xl` machine type